### PR TITLE
Autom/GLA_BSA-5_BSA-1

### PR DESCRIPTION
**Pull request by LanderoX144**
…cion de estos datos en edit

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "module": "pytest",
       "args": [
         "-k",
-        "test_metod_name",
+        "test_BSA_1_GLA",
```
Comment (eduxsan) [inline .vscode/launch.json:14]: Please revert this change

File: Tests/New_Customer/test_new_customer.py [hunk 2]
```diff
@@ -23,3 +23,33 @@ def test_new_customer_input_message(self):
...
+        addCustomer.write_on_input("Phone","5599345690")
+        addCustomer.write_on_input("email","Hola@test.com")
+        addCustomer.click_on_input("Submit")
+        
+
+
+        
+        
+        
+
```
Comment (eduxsan) [inline Tests/New_Customer/test_new_customer.py]: Please remove this empty lines

**Review by eduxsan — CHANGES_REQUESTED** [review]

File: Pages/new_customer_page.py [hunk 3]
```diff
@@ -33,3 +61,161 @@ def verify_message_by_input_name(self, input_name, expected_message):
...
+            elif  input_name == "Email":
+                element = self.driver.find_element(*NewCustomerLocators.email_input_message)
+            elif  input_name == "Address":
+                element = self.driver.find_element(*NewCustomerLocators.customer_address_input)
+        except Exception as e: 
+                pass
+    def verify_message_by_input_name(self, input_name, expected_message):
+        pass
+        
+
```
Comment (eduxsan) [inline Pages/new_customer_page.py]: Please remove this

File: Tests/New_Customer/test_new_customer.py [hunk 4]
```diff
@@ -20,6 +23,157 @@ def test_new_customer_input_message(self):
...
+        editCustomer.verify_data_by_field("Address", "Calle Bosques de Mexico 32")
+        editCustomer.verify_data_by_field("City","CiudaddeMexico")
+        editCustomer.verify_data_by_field("State","CDMX")
+        editCustomer.verify_data_by_field("Pin","548320")
+        editCustomer.verify_data_by_field("Phone","5599345690")
+        
+    
+        
+        
+
```
Comment (eduxsan) [inline Tests/New_Customer/test_new_customer.py]: Please remov ethis

**Review by eduxsan — CHANGES_REQUESTED** [review]

**Review by eduxsan — APPROVED** [review]

File: Locators/locators.py [hunk 5]
```diff
@@ -16,8 +16,52 @@ class LoginLocators:
...
+        customer_gender_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[5]/td[2]/input[1]")
+        customer_date_birth_input =(By.CSS_SELECTOR, "#dob")
+        customer_address_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[7]/td[2]/textarea") 
+        customer_city_input = (By.XPATH,  "/html/body/table/tbody/tr/td/table/tbody/tr[8]/td[2]/input") 
+        customer_state_input = (By.XPATH,"/html/body/table/tbody/tr/td/table/tbody/tr[9]/td[2]/input")
+        customer_pin_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[10]/td[2]/input")
+        customer_telephone_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[11]/td[2]/input")
+        customer_email_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[12]/td[2]/input")
+        submit_add_customer_button = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[13]/td[2]/input[1]")
+        id_customer_reg_msg = (By.XPATH, '//*[@id="customer"]/tbody/tr[4]/td[2]')
```
Comment (AlanDiCha123) [inline Locators/locators.py]: Check your identation

File: Pages/new_customer_page.py [hunk 6]
```diff
@@ -14,16 +15,43 @@ def click_on_input(self, input_name):
                 element = self.driver.find_element(*NewCustomerLocators.customer_name_txt)
```
Comment (AlanDiCha123) [inline Pages/new_customer_page.py:15]: Where is your "if" statement?

File: Pages/new_customer_page.py [hunk 7]
```diff
@@ -14,16 +15,43 @@ def click_on_input(self, input_name):
...
             print(e)
+    
+    def click_on_submit(self):
+        try:
+            self.driver.find_element(*NewCustomerLocators.submit_button).click()
+        except Exception as e:
+            print(e)
 
     def verify_message_by_input_name(self, input_name, expected_message):
         try:
```
Comment (AlanDiCha123) [inline Pages/new_customer_page.py:40]: The "except" is missing

File: Tests/New_Customer/test_new_customer.py [hunk 8]
```diff
@@ -1,14 +1,17 @@
+from Pages.edit_customer_page import EditCustomerPage
 from Pages.login_page import LoginPage
-from Pages.home_page import HomePage
 from Pages.common_page import CommonPage
 from Pages.new_customer_page import NewCustomerPage
 import pytest
 import time
+from Pages.edit_customer import EditCustomerPage
```
Comment (AlanDiCha123) [inline Tests/New_Customer/test_new_customer.py:7]: Import is used twice

**Review by AlanDiCha123 — CHANGES_REQUESTED** [review]
Check your code

File: Pages/new_customer_page.py (hunk 6 above)
Comment (AlanDiCha123) [reply Pages/new_customer_page.py:15]: Sorry

File: Pages/new_customer_page.py (hunk 7 above)
Comment (AlanDiCha123) [reply Pages/new_customer_page.py:40]: Sorry

File: Tests/New_Customer/test_new_customer.py (hunk 8 above)
Comment (AlanDiCha123) [reply Tests/New_Customer/test_new_customer.py:7]: Sorry

**Review by AlanDiCha123 — APPROVED** [review]
Well done

**Pull request merged**